liferay-gradle/blade.controlmenuentry/build.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
-	compile group: "com.liferay", name: "com.liferay.product.navigation.control.menu.api", version: "2.0.0"
+	compile group: "com.liferay", name: "com.liferay.product.navigation.control.menu.api", version: "[3.0.0-SNAPSHOT,3.0.0)"
```

Comment:
Hey @juliocamarero i don't see 3.0.0 available here:
https://repository.liferay.com/nexus/content/groups/public/com/liferay/com.liferay.product.navigation.control.menu.api/

So I hesitate to update this sample until that version is available.  what do you think?